etc/kayobe/inventory/host_vars/ethsw-b17-u40-41

```diff
@@ -59,6 +61,42 @@ switch_interface_config:
     config: "{{ switch_interface_config_all }}"
   Te1/1/3/4:
     config: "{{ switch_interface_config_all }}"
+  Fo1/4/4/1:
+    description: ethsw-b17-u42-Fo1/30
+    config: "{{ switch_interface_config_trunk }}"
+  # 4x10G LAG to SoftIron 10GBaseT Netgear switch
+  Te1/4/5/1:
+    description: softiron-switch-lag-xg25
+    config: >
+      {{ switch_interface_config_lag_member +
+         ['port-channel-protocol lacp',
+          ' port-channel 2 mode active',
+          ' exit'] }}
+  Te1/4/5/2:
+    description: softiron-switch-lag-xg26
+    config: >
+      {{ switch_interface_config_lag_member +
+         ['port-channel-protocol lacp',
+          ' port-channel 2 mode active',
+          ' exit'] }}
+  Te1/4/5/3:
+    description: softiron-switch-lag-xg27
+    config: >
+      {{ switch_interface_config_lag_member +
+         ['port-channel-protocol lacp',
+          ' port-channel 2 mode active',
+          ' exit'] }}
+  Te1/4/5/4:
+    description: softiron-switch-lag-xg28
+    config: >
+      {{ switch_interface_config_lag_member +
+         ['port-channel-protocol lacp',
+          ' port-channel 2 mode active',
+          ' exit'] }}
+  # SoftIron/Ceph Storage network link to dev cluster
+  Hu1/4/6:
```

Comment:
Oh yes - ethsw1-ra12 Eth 1/11 is a 100G link between dev and alaska for Ceph (VLAN 4)